php_phongo.c

```diff
@@ -1289,6 +1289,7 @@ static bool php_phongo_apply_wc_options_to_uri(mongoc_uri_t *uri, bson_t *option
 	return true;
 } /* }}} */
 
+#ifdef MONGOC_ENABLE_SSL
```

Comment:
#404 used `#if MONGOC_ENABLE_SSL`, but `mongoc-config.h` will `#undef` the constant if is not equal to `1`.